### DISCUSSION TITLE
Onion Skin Issue with duplicated layer

### DISCUSSION THREAD

**Discussion by spaceoutlawed** [keiwando/toonsquid#20] [Q&A]
i was creating an animation and duplicated my lineart layer so i could have an easier time coloring, however, despite this practice working moments prior with a different section of the animation, i found the onion skin suddenly stopped working. another issue i found was my brush stokes not showing up whenever i drew, this is only when onion skin is enabled, which i need to have so i know where to put the markings on the character. this issue isn’t shown whenever i playback the animation, but whenever i am on the selected coloring layer, it appears invisible despite having color to it. 

any help would be appreciated! examples (should be) attached, im very new to github so apologies if the images aren’t correctly attached. only solution i’ve found is coloring on the lineart layer which i am very hesitant to fully commit to if there is a solution to this 

(note; the circle in the back is not affected by any of this i just forgot to disable it)

![0C68C6C5-32F5-4C07-9C27-43F36EED873B](https://user-images.githubusercontent.com/115650856/195390314-1a6293a8-733a-4621-b193-6ced09a648c6.jpeg)
![C6CA146D-B67B-4A6B-A771-FA7E16039556](https://user-images.githubusercontent.com/115650856/195390319-3cbeaf5f-216f-45a2-9a79-d0cc41ce7254.jpeg)
![C04E47DB-5F92-4A36-BC9F-6DF0A0A495C9](https://user-images.githubusercontent.com/115650856/195390321-ac200c1e-dfd2-4174-bfeb-c226fe8a33e4.jpeg)
glitchy sped up playback 
![3060B232-1619-43F2-A47C-666B5F440F00](https://user-images.githubusercontent.com/115650856/195390322-1c5cbf0c-c909-4bda-94f5-9dddbce480ef.gif)

**Comment by spaceoutlawed** [comment]
i ended up figuring out a solution, duplicating the lineart layer once more and using that to color instead, any reason as to why this may have happened is still very much appreciated

**Comment by keiwando** [comment]
Hi! Thanks for letting me know about this. Would it be possible for you to export the project and send it to me so that I can take a look at your exact situation? You can send it to me via contact@toonsquid.com as well if you don't want to upload it here publicly.

Apologies for the inconvenience!

**Reply by spaceoutlawed** [reply]
i tried emailing it and attaching it here and it gave me an error code due to the project file type, is there a specific way i should convert it? it's 14MB

edit: think i figured out how to email it

**Reply by keiwando** [reply]
Thank you, that is very helpful! I can reproduce the problem with your project and will let you know once an update is ready that fixes this bug.

**Reply by spaceoutlawed** [reply]
no problem!; and sorry for any inconveniences! ^^

**Reply by keiwando** [reply]
You have nothing to apologize for!

It seems to be related to the fact that Layer 9 is set to clip to Layer 1. It's still a bug of course and shouldn't happen but maybe you can temporarily disable the clipping until I have the update with the fix ready.

**Reply by spaceoutlawed** [reply]
disabling the clipping worked! thank you so much!

**Comment by keiwando** [comment]
The 1.5.2 update is now available and should fix this issue.
